rollup.config.js

```diff
@@ -116,4 +116,52 @@ export default [
       sourcemap: true
     }
   },
+  {
+    input: 'static/js/publisher/form/multiselect.js',
```

Comment:
Do we need it as separate file? This will make it bit harder when we finally migrate to webpack and also it uses react, so it means that users will download react multiple times (in each bundle that uses it).

Reply:
Good point, updated to include it as part of 'publisher.js' and included the relevant bits in the config.